CONTRIBUTING.md

```diff
@@ -385,6 +386,60 @@ tests for [Redux reducers](test/reducer_test.dart) and a few tests for
 straightforward to make, adding React components and Redux middleware
 unit tests is an [ongoing issue](https://github.com/UC-Davis-molecular-computing/scadnano/issues/32).
 
+### Formatting Dart code
+
+Before making a Pull Request, all the dart code must be formatted using one of the below given options. If this step is not done, the formatting check for the Pull Request will fail.
```

Comment:
Maybe add a comment saying why we do this: so that PR's will only show "substantial semantic" changes, not minor whitespace changes, helping to review code changes more quickly by focusing only on changes that actually affect the code logic.